.github/workflows/release.yml

```diff
@@ -0,0 +1,93 @@
+name: Publish Docker Images
+
+on:
+  push:
+    tags:
+      - 'kpi-tracker/*.*.*'
+      - 'recover-id-object/*.*.*'
+      - 'wallet-connect-test-bench/*.*.*'
+
+env:
+  REGISTRY: docker.io
+
+jobs:
+  publish-docker-image:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - IMAGE_NAME: 'kpi-tracker'
+            CONTEXT_DIR: 'kpi-tracker'
+            DOCKERFILE_PATH: 'scripts/Dockerfile'
+            PROJECT_TYPE: 'rust'
+            ENVIRONMENT: 'testnet-deployments'
+
+          - IMAGE_NAME: 'recover-id-object'
+            CONTEXT_DIR: 'recover-id-object'
+            DOCKERFILE_PATH: 'scripts/Dockerfile'
+            BUILD_ARGS: ['BUILD_IMAGE=rust:1.74-buster']
+            PROJECT_TYPE: 'rust'
+            ENVIRONMENT: 'release-recover-id-object'
+
+          - IMAGE_NAME: 'wallet-connect-test-bench'
+            CONTEXT_DIR: 'wallet-connect-test-bench/front-end'
+            DOCKERFILE_PATH: 'Dockerfile'
+            PROJECT_TYPE: 'frontend'
+            ENVIRONMENT: 'testnet-deployments'
+
+    if: contains(github.ref_name, matrix.IMAGE_NAME)
```

Comment:
Should not use contains, but instead check the entire name as we also do elsewhere